local_pathfinding/node_navigate.py

```diff
@@ -176,7 +178,22 @@ def get_desired_heading(self) -> float:
         )
 
         # TODO: create function to compute the heading from current position to next local waypoint
-        return 0.0
+        # Find index of self.gps in current waypoints
+        current_waypoint = self.gps.lat_lon
+        next_waypoint = None
+        waypoints = self.local_path.waypoints
+        for i in range(len(waypoints)):
+            if waypoints[i] == current_waypoint and i != len(waypoints) - 1:
+                next_waypoint = waypoints[i+1]
+                break
+        if next_waypoint is None:
+            return 0.0
+        heading = math.atan2(
+            (next_waypoint.longitude-current_waypoint.longitude) *
+            math.cos(current_waypoint.latitude),
+            (next_waypoint.latitude-current_waypoint.latitude)
+        )
+        return heading
```

Comment:
I think it looks correct, but I just wanted to double check that when it gets published, `desired_heading` follows the conventions in custom interfaces? 
https://github.com/UBCSailbot/custom_interfaces/blob/main/msg/DesiredHeading.msg
https://github.com/UBCSailbot/custom_interfaces/blob/main/msg/HelperHeading.msg